agenta-backend/agenta_backend/services/organization_service.py

```diff
@@ -1,9 +1,31 @@
+import os
+import sendgrid
+from typing import List
 from bson import ObjectId
-from agenta_backend.services.db_manager import engine, OrganizationDB
+from datetime import datetime
+from fastapi import HTTPException
+from sendgrid.helpers.mail import Mail
 from agenta_backend.models.api.organization_models import (
+    User,
     Organization,
     OrganizationUpdate,
 )
+from agenta_backend.models.db_models import OrganizationDB
+from agenta_backend.models.db_engine import DBEngine
+
+# Initialize database engine
+engine = DBEngine(mode="default").engine()
+
+sg = sendgrid.SendGridAPIClient(api_key=os.environ.get("SENDGRID_API_KEY"))
+
+
+async def check_user_org_access(kwargs: dict, organisation_id: ObjectId) -> bool:
+    user_organisations: List = kwargs["organization_ids"]
```

Comment:
Can you perform a global rename refactoring for "organisations" to "organizations"?